src/Styleguide/Components/Collapse.tsx

```diff
@@ -1,50 +1,24 @@
 import React from "react"
-import styled from "styled-components"
+import { animated, Spring } from "react-spring"
```

Comment:
😄

Reply:
Since we had all of that back and forth in the other PR that introduced `<Collapse>`, it might be good to resurface that discussion.

Reply:
Yeah was going to ask about that too-- for reference it is here: https://github.com/artsy/reaction/pull/1114#discussion_r209354107